### PR TITLE
Adde Redux in React components

### PR DESCRIPTION
**Pull request by Dmambo**
## Features Added

Displayed books from the Redux store.

- [ ] Use `useSelector` to consume the state.
- [ ] Import Redux `useDispatch`.
- [ ] Set the data inputs in the local React state set the title and author .
- [ ] Takes the inputs from the state and generates a unique id and add them to an object.
- [ ] Once your new book object is ready to be submitted to Redux store, dispatch a corresponding action
- [ ] Dispatch a corresponding action

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Dmambo,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly write more comprehensive documentation. This project is Bookstore, but your README contains information about Math-magician.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -26,19 +26,19 @@
 
 <!-- PROJECT DESCRIPTION -->
```
Comment (TedLivist) [inline README.md:28]: - [ ] Kindly implement the action dispatch for displaying "Under construction" on the Category page. When I click the button I should see the text.

File: README.md [hunk 2]
```diff
@@ -26,19 +26,19 @@
...
 
-# 📖 MATH MAGICIAN <a name="about-project"></a>
+# 📖 BOOK STORE <a name="about-project"></a>
```
Comment (TedLivist) [inline README.md:29]: - [ ] The following sections in your README contain dummy text from the template: Usage, Install, Setup, FAQ (optional - can be removed). Kindly update your sections to be more descriptive.

- [ ] Kindly add the Future Features section in your README to tally with the link in the Table of contents.

- [ ] Your Key Features section still contains information about Math-magician. Kindly fix that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Dmambo,

Good job so far!
✔️ Use of local state for input
✔️ Books can be added and removed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Dmambo ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**